### ISSUE TITLE
Toolbar UI is broken

### ISSUE THREAD

**Issue by greenchapter** [brunophilipe/Cakebrew#150]
It seems the Toolbar UI is broken. Some Icons are missing and the translation text seems to be currupted.
<img width="622" alt="bildschirmfoto 2015-10-14 um 23 59 11" src="https://cloud.githubusercontent.com/assets/693276/10500209/e6e754d4-72d6-11e5-8b7a-32adfd992c8b.png">

**Comment by brunophilipe** [comment]
Thanks for reporting the bug. I won't have time to check that now. Please download the latest version from the Cakebrew website if the updated one doesn't work.

If after that it still won't work, please try going into the Cakebrew bundle and moving the "de.lproj" folder to the trash to force the app to run in English.

**Comment by davidspiess** [comment]
Same problem with german language

**Comment by brunophilipe** [comment]
Version of OS X, please? I am definitely not able to reproduce this in 10.11.0.

**Comment by davidspiess** [comment]
<img width="1052" alt="bildschirmfoto 2015-10-16 um 21 28 32" src="https://cloud.githubusercontent.com/assets/992674/10551400/feada704-744c-11e5-98b1-eda2447235b4.png">
I also tried to remove it completely and install again :)

**Comment by brunophilipe** [comment]
I feel ashamed now because I was now able to reproduce it. I was definitely looking at the wrong things.
Anyway, these three strings are missing translations in German. Would you mind supplying them?

```
"Toolbar_More_Information" = "More Information";
"Toolbar_Homebrew_Update" = "Update Homebrew";
"Toolbar_Search" = "Search Formulae";
```

**Comment by davidspiess** [comment]
Here you go :) :+1: 
```
"Toolbar_More_Information" = "Weitere Informationen";
"Toolbar_Homebrew_Update" = "Homebrew aktualisieren";
"Toolbar_Search" = "Formeln durchsuchen";```

**Comment by brunophilipe** [comment]
Fixed in commit 665a0864b3f1524371e7faa6580bed222c337323

Will release a new version in moments
